### PR TITLE
chore(blog): add case study link to Terralith article

### PR DESCRIPTION
**Pull request by oycyc**
Previous line is "Stay tuned for one of our upcoming post" - so this adds it!

## Summary by CodeRabbit

- **New Features**
	- Added a new section to the Terralith Monolithic Terraform Architecture document, featuring a case study on managing a large Terralith with over 43,000 resources.
  
- **Documentation**
	- Enhanced content with practical insights on challenges and strategies related to Terralith architecture.

### CODE REVIEW

**Comment by oycyc** [comment]
This is what it looks like on the preview: 
image

File: content/updates/terralith-monolithic-terraform-architecture.md [hunk 1]
```diff
@@ -104,3 +104,5 @@ Ultimately, the key is to have a scalable strategy where the goal is to create a
 By understanding the pitfalls of the Terralith architecture, you and your teams can avoid mistakes that will lead to further issues.
 
 Stay tuned for one of our upcoming post featuring practical tips on breaking down a Terralith, as well as a case study detailing how we helped a client decompose their Terralith to achieve a more scalable and maintainable infrastructure!
+
+_12/2/2024 Update:_ We've published the case study where we decomposed a 43,000+ resource Terralith. [Take a read here!](https://masterpoint.io/power-digital-case-study/)
```
Comment (oycyc) [inline content/updates/terralith-monolithic-terraform-architecture.md:108]:
```suggestion
_**12/2/2024 Update:** We've published the case study where we decomposed a 43,000+ resource Terralith. [Take a read here!](https://masterpoint.io/power-digital-case-study/)_
```

**Comment by oycyc** [comment]
This is what it looks like on the preview: 
image

**Review by Gowiem — APPROVED** [review]
Great stuff @oycyc -- Thanks for following up!

**Pull request merged**